packages/web/src/components/search/SearchBox.js

```diff
@@ -1812,8 +1834,8 @@ const SearchBox = (props) => {
 													)}
 													{renderError(true)}
 												</SearchBoxAISection>
-											)}
-											{!showAIScreen && (
+											) : null}
+											{!showAIScreen && hasSuggestions() && !hasCustomRenderer(props) ? (
```

Comment:
@SavvyShah please explain the code change.

Reply:
I moved it back, which would also fix the regression